ansible/webgoat.sh

```diff
@@ -1,7 +1 @@
-java -javaagent:contrast.jar \
--Dcontrast.dir=working \
--Dcontrast.server=demo \
--Dcontrast.server.activity.period=5000 \
--Dcontrast.log.daily=true \
--Dcontrast.level=info \
--jar webgoat-container-7.0.1-war-exec.jar
+java -javaagent:contrast.jar -Dcontrast.dir=working -Dcontrast.server=PROD-DEMO -Dcontrast.server.activity.period=5000 -Dcontrast.log.daily=true -Dcontrast.level=info -jar webgoat-container-7.0.1-war-exec.jar
```

Comment:
We've seen some confusing behavior with our CI plugins unless we set `contrast.path` in the jvm args. Up to you if you want to add it or not.